src/adapters/poi/specials/navigator_geolocalisation_poi.js

```diff
@@ -24,7 +24,11 @@ export default class NavigatorGeolocalisationPoi extends Poi {
   }
 
   async geolocate() {
-    await GeolocationCheck.checkPrompt();
+    const isAuthorized = await new Promise(resolve =>
+      GeolocationCheck.checkPrompt(isAuthorized => resolve(isAuthorized))
+    );
+    if (!isAuthorized) {return;}
```

Comment:
You are changing the signature of the function by not returning a rejecting Promise here. It may disturb other parts of the app relying on that. https://github.com/QwantResearch/erdapfel/blob/master/src/panel/direction/DirectionInput.jsx#L63-L72